src/pages/index.tsx

```diff
@@ -153,12 +145,8 @@ const Home: NextPage<any> = () => {
         return (
           <RequestSidebar
             currentUser={user}
-            sent={toRequestIds
-              .map(convertToUser)
-              .filter((user): user is PublicUser => !!user)}
-            received={fromRequestIds
-              .map(convertToUser)
-              .filter((user): user is PublicUser => !!user)}
+            sent={requests?.from.map((req) => req.toUser!) ?? []}
+            received={requests?.to.map((req) => req.fromUser!) ?? []}
```

Comment:
I can get rid of the forced null unwrap here later so we dont have bad crashes, I fixed the type